packages/content/src/content.ts

```diff
@@ -29,6 +30,12 @@ export function getContent(
       return Promise.reject(e);
     });
   }
+  return ret.then(content => applyNormalizedType(content));
```

Comment:
this should be done in `itemToContent()`, not here

Reply:
Won't that make it so that it's only attached when we're using the portal API?

Reply:
nope, `itemToContent()` is called by `datasetToContent()` as well

Reply:
Nevermind. I see